### DISCUSSION TITLE
MSI windows installer for 32 bits is discontinued

### DISCUSSION THREAD

**Discussion by g-bougard** [glpi-project/glpi-agent#570] [Announcements]
### Announcement

As announced in [GLPI Agent 1.7 release blog post](https://glpi-project.org/glpi-agent-1-7/), we decided to no more provide 32 bits version for future MSI windows installer.

Starting from v1.8, only MSI windows installer for 64 bits will be provided.

Please comment here if you have any concern with this announcement.

**Comment by czare** [comment]
Some environments may still employ Windows Server 2003 machines, which is not 64x compatible, and GLPI native inventory is only availabe at version 10 onwards, or something like that, right? If I install an 1.7.2 Agent version, will it be possible to use the native inventory function from GLPI verison 10?

**Reply by g-bougard** [reply]
Don't worry about GLPI 10 native inventory support, even FIA is supported by GLPI 10 native inventory. The feature doesn't depend on agent and even less on its architecture installation.

For 32 bits version, please use [1.7.3](https://github.com/glpi-project/glpi-agent/releases/tag/1.7.3) version, this is the last one which supports 32 bits installer.
